test/install_tests.sql

```diff
@@ -12,6 +12,7 @@
 @core.pkb
 @ut_utils/test_ut_utils.pkb
 @ut_annotations/test_annotations.pkb
+show errors
```

Comment:
Doesn't work at the end of the script (there is already one) - but I can remove it again of course.